.vscode/README.md

```diff
@@ -22,49 +22,100 @@ Install the following VSCode extensions:
 
-* Press `Ctrl + Shift + P` in VSCode.
-* Type `Tasks`.
-* Select `Run Task` (press Enter).
-* Select the suitable task for your situation.
+### Dependencies
 
+Required and optional dependencies for modflow6 are discussed in [DEVELOPER.md](../DEVELOPER.md)
 
-### Language Server
-
-1. Install the fortran language server via:
+The Modern Fortran extension requires a Fortran compiler and language server.  To install
+the [fortls](https://github.com/gnikit/fortls) fortran language server run:
 
 ```bash
 pip install -U fortls
 ```
 
-2. Find out where the executable is
+The vscode-modern-fortran-formatter requires `fprettify`, which can be installed in a similar way
+using pip or conda.
+
 
-- bash: `which fortls`
-- cmd: `where fortls`
-- PowerShell: `Get-Command fortls`
+### Settings
 
-3. Add the setting "fortran.fortls.path" to `.vscode/settings.json` and set its value to the result of step 2.
-In case this file does not exist yet, create a new one.
+Add [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) to the
+`modflow6/.vscode` directory if not already there. The following settings can be considered a
+starting place as the contents of this file are dictated both by desired VSCode behavior and
+environmental factors:
 
+In general, to determine the path of an installed tool in your environment run:
+- bash: `which <toolname>`, e.g. `which fortls`
+- cmd: `where <toolname>`, e.g. `where python`
+- PowerShell: `Get-Command <toolname>` e.g. `Get-Command fprettify`
 
+The setting "python.defaultInterpreterPath" points toward your Python executable:
+```json
+{
+    "python.defaultInterpreterPath": "/path/to/python",
+}
+```
+
+The setting "fortran.fortls.path" points toward your fortls executable:
 ```json
 {
     "fortran.fortls.path": "/path/to/fortls"
 }
 ```
 
+The fortran formatter can be integrated with vscode using the following settings:
+
+```json
+{
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "yukiuuh.vscode-modern-fortran-formatter",
+    "modernFortranFormatter.fprettifyArgs": "-c C:\\<full path to modflow6>\\distribution\\.fprettify.yaml"
+}
+```
+
+...or, in a WSL environment:
+
+```json
+{
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "yukiuuh.vscode-modern-fortran-formatter",
+    "modernFortranFormatter.fprettifyArgs": "-c /<full path to modflow6>/distribution/.fprettify.yaml",
+}
+```
+
+Setting the formatter up in this way will cause a source file to reformat with each explicit save.
+
+### Running VSCode
+
+Open the top level `modflow6` repository directoy on your system when starting VSCode. The program will
+then look for the `modflow6/.vscode` directory to discover settings relevant to your session.
+
+A nice alternative on any system is to start VSCode from the shell. For example, in a bash or git bash
+shell (windows), change to the `modflow6` directory and execute the command:
```

Comment:
On windows I would recommend powershell

Reply:
I wasn't intending to make a recommendation but rather to provide an example.  Let me know what you think should be communicated and I can make that change.